platform/docs/docs/platform/services/ui/customization-service.md

```diff
@@ -275,6 +275,50 @@ const overlayItem: Types.UIOverlayItem = {
 
 This section can be used to specify various customization capabilities.
 
+## Context Menus
+The context menu customization is a relatively complex one, with two layers.
+The first layer is the choice of the command to run to display the context menu.
+For the case of the cornerstone viewer context menu, this is specified by a
+customization called `showViewerContextMenu`.  Then, the actual menu is chosen
+by the menuName as specified in the showViewerContextMenu, OR by the actual
+items list in thew viewer context menu.
+
+### Show Context Menu Customization
+The command to show the viewer context menu can be overridden in order to
+show alternate dialogs on right click/context menu views, for example,
+to show the label dialog automatically for the arrow annotation, or to show a
+different type of context menu, such as a circular menu.  The example below
+shows the configuration for that:
+
+```js
+   customizationService.addCustomizations([{
+    id: 'showContextMenu',
+    commands: [
+      {
+        commandName: 'showCircularContextMenu',
+        commandOptions: {
+          menuName: 'cornerstoneContextMenu',
```

Comment:
I'm not sure If I like `cornerstoneContextMenu` or just `contextMenu`, including cornerstone in any name should be releated to rendering side of things (or tools), the context menu stuff is purely an app concern and having the library name in the names and ids probably confuses users